src/PolykeyAgent.ts

```diff
@@ -583,7 +584,7 @@ class PolykeyAgent {
       await this.sessionManager.start({ fresh });
       await this.status.finishStart({
         pid: process.pid,
-        nodeId: this.keyManager.getNodeId(),
+        nodeIdEncoded: nodesUtils.encodeNodeId(this.keyManager.getNodeId()),
```

Comment:
Same here, we should still be calling it `nodeId` on the outside world. The fact that it is encoded as a value is an implementation detail.

To the outside world, node ids are encoded. Internally node ids are buffers. That's how it should be understood.

Reply:
This has also been changed back.